### ISSUE TITLE
Project is active?

### ISSUE THREAD

**Issue by mshalmanov** [bpellin/keepassdroid#469]
Hello. This is project is active?

**Comment by Nickoriginal** [comment]
Yes, it is. Perhaps, sometimes it doesn't seems to be so, because it develops slowly and almost doesn't include any new functions but it is not abandoned.

**Comment by mshalmanov** [comment]
Hello. May i can help to develop this project ?

**Comment by Nickoriginal** [comment]
Yes, of course. You can open the issue about the problem in the app, suggest a new feature or open a pull request. Read more about this at https://opensource.guide/

**Comment by mshalmanov** [comment]
Where i can see build and config develop workspace manuals for begin help ?

**Comment by Nickoriginal** [comment]
This repository doesn't contain them, you can seek them manually or see some useful tips in the [KeePassDX](https://github.com/Kunzisoft/KeePassDX) and [Keepass2Android](https://github.com/PhilippC/keepass2android) related apps repositories, that can help in developing this app too.

**Comment by mshalmanov** [comment]
Hello. I fork you app and config my workspace. How i can begin start to build and run this app ? how can i contact you?

**Comment by Nickoriginal** [comment]
The app's owner and main contributor is @bpellin, you can contact him and wait a bit for answer.

**Comment by mshalmanov** [comment]
@bpellin Hello. I fork you app and config my workspace. How i can begin start to build and run this app ? how can i contact you?

**Comment by Nickoriginal** [comment]
Hm... The latest commit is dated by June, and seems to be pull requests no longer accepted (at least, now). If you want, you can look into KeePassDX or Keepass2Android issues, they are more actively developed.
